src/Home.py

```diff
@@ -51,114 +53,139 @@ def get_df():
+st.markdown("This migration data is more than just numbers. This is information about real people that left their home countries for various reasons and ultimately perished during their journey. This is data that requires respect and care in order to be presented faithfully. In this project, we aim to bring awareness to the challenges and risks that migrants and their loved ones face everyday. Perhaps you are familiar with several of the migration routes displayed here. There are likely other dangerous routes that you haven't heard of. Many of us can recall certain stories that made it to the news about the deaths of migrants, but as we can see in the data, this is an ongoing, everyday risk that affects many individuals, families, communitites, and countries around the world.")
 st.markdown("We welcome you to explore the below visualization by zooming into the map directly or by using the filters in the left side menu.")
 
-#map_w = folium.Map()
-#loca = (migrantdf.iloc[0]['lon'],migrantdf.iloc[0]['lat'])
-#custom_marker = folium.Marker(location=loca)
-#custom_marker.add_to(map_w)
-#st_map = st_folium(map_w)
-# 1.2 map
-
-#migrantdf['text'] = migrantdf['Cause of Death Abbreviation'] + migrantdf['Total Number of Dead and Missing'].to_string()
+# 1.2 Global Statistics
+# st.markdown(str(year_input), str(len(year_input)))
+if len(year_input) == 1:
+    year_text = " in " + str(year_input[0])
+elif len(year_input) == 2:
+    year_text = " in " + str(year_input[0]) + ' and ' + str(year_input[1])
+elif len(year_input) > 2:
+    year_text = " in " + str(year_input[0])
+    for year in year_input[1:-1]:
+        year_text += ', ' + str(year)
+    year_text += ', and ' + str(year_input[-1])
+else:
+    year_text = ''
 
+if len(year_input) > 0:
+    statsdf = df[df['Incident year'].isin(
+        year_input)]
+else:
+    statsdf = df
+df_cause_of_death = statsdf.groupby('Cause of Death Abbreviation')['Cause of Death Abbreviation', 'Total Number of Dead and Missing'].sum(
+).sort_values('Total Number of Dead and Missing', ascending=False).reset_index()
+df_cause_of_death['Total Number of Dead and Missing'] = df_cause_of_death['Total Number of Dead and Missing'].apply(
+    lambda x: prettify(x))
+
+st.subheader(f'Global Statistics' + str(year_text))
+st.markdown(f'Most common causes of death')
+st.dataframe(df_cause_of_death, height=200)
+
+col2, col3 = st.columns(2)
+with col2:
+    st.metric(f'Total number of Recorded Dead and Missing',
+              prettify(statsdf['Total Number of Dead and Missing'].sum()))
+with col3:
+    st.metric(f'Total Number of Recorded Survivors',
+              prettify(statsdf['Number of Survivors'].sum()))
+
```

Comment:
Added in these global statistics, with the ability to filter through via year:
<img width="1399" alt="Screen Shot 2022-10-22 at 11 12 06" src="https://user-images.githubusercontent.com/53492091/197328678-083a4d6d-88b9-415d-a9eb-41c0db1ba6f3.png">